cmake/thisdd4hep.sh

```diff
@@ -99,22 +98,19 @@ if [ ${Geant4_DIR} ]; then
     test -r ${G4ENV_INIT} && { cd $(dirname ${G4ENV_INIT}) ; . ./$(basename ${G4ENV_INIT}) ; cd $OLDPWD ; }
     #---- if geant4 was built with external CLHEP we have to extend the dynamic search path
     if [ @GEANT4_USE_CLHEP@ ] ; then
-	dd4hep_add_library_path @CLHEP_LIBRARY_PATH@;
+        dd4hep_add_library_path @CLHEP_LIBRARY_PATH@;
     fi;
     export CLHEP_DIR=@CLHEP_INCLUDE_DIR@/../;
     export CLHEP_ROOT_DIR=@CLHEP_INCLUDE_DIR@/../;
     export CLHEP_INCLUDE_DIR=@CLHEP_INCLUDE_DIR@;
-    dd4hep_add_library_path ${G4LIB_DIR};
     unset G4ENV_INIT;
     unset G4LIB_DIR;
 fi;
 #
 #----XercesC LIBRARY_PATH-----------------------------------------------------
 if [ ${XERCESCINSTALL} ]; then
-    #dd4hep_add_path    PATH ${XERCESCINSTALL}/bin;
     dd4hep_add_library_path ${XERCESCINSTALL}/lib;
 fi;
-dd4hep_add_library_path @BOOST_DIR@
```

Comment:
Is this because BOOST_DIR is not actually pointing at a library folder?

Reply:
Yes, in this current form it never made sense.

Reply:
OK, then it is fine.